plugins/aea-test-autonomy/aea_test_autonomy/docker/acn_node.py

```diff
@@ -68,7 +68,7 @@ def __init__(
     @property
     def tag(self) -> str:
         """Get the image tag."""
-        return "valory/acn-node:0.1.0"
+        return "valory/acn-node:latest"
```

Comment:
changed to `latest`, because that is what we use for all our images (and 0.1.0 is currently latest)